runtime/plugins/comment/help/comment.md

```diff
@@ -80,18 +80,23 @@ but it is only available for certain filetypes:
 * zsh: `# %s`
 
 If your filetype is not available here, you can simply modify
-the `commenttype` option:
+the `comment.type` option:
 
 ```
-set commenttype "/* %s */"
+set comment.type "/* %s */"
 ```
 
 Or in your `settings.json`:
 
 ```json
 {
     "*.c": {
-        "commenttype": "/* %s */"
+        "comment.type": "/* %s */"
     }
 }
 ```
+
+`commenttype` (without the dot) is the legacy option that is
+superseded by `comment.type`. `commenttype` is still supported
+but will get deprecated in the future.
```

Comment:
@Neko-Box-Coder:
As mentioned [here](https://github.com/zyedidia/micro/pull/3424#discussion_r2106326952), can you adapt it to something like:
```suggestion
superseded by `comment.type`. `commenttype` is still supported
but deprecated from now on.
```